src/schematools/types.py

```diff
@@ -748,6 +748,14 @@ def dataset(self) -> DatasetSchema:
         """The dataset that this table is part of."""
         return self.parent_schema
 
+    @property
+    def parent_table_field(self) -> Optional[DatasetFieldSchema]:
+        """Provide the NM-relation that generated this through table."""
+        if self.through_table or self.nested_table:
+            return self._parent_table.get_field_by_id(self["originalID"])
```

Comment:
Be carefull to check on this. @gkoller changed `_parent_table` to `parent_table`.

Reply:
Yes, this checks the internal field, which is fine. But the outside world should use the public property off course.